src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/SpaceManagementTest.java

```diff
@@ -120,27 +118,34 @@ void setupKernel(Path storeDirectory)
     @Test
     void GIVEN_user_component_config_with_space_management_WHEN_space_exceeds_THEN_excess_log_files_are_deleted()
             throws Exception {
-        tempDirectoryPath = Files.createDirectory(tempRootDir.resolve("IntegrationTestsTemporaryLogFiles"));
```

Comment:
I agree this is not the best test, but my goal was just to make what we had not flaky